requirements.txt

```diff
@@ -1,7 +1,8 @@
 pip~=22.0
 ninja
 cmake
-cuquantum>=22.3.0
+custatevec-cu11
+mpi4py
```

Comment:
```suggestion
```
Since this dependency is optional, let's leave it out

Reply:
Sure!